chunky/src/java/se/llbit/chunky/ui/render/SkyTab.java

```diff
@@ -81,12 +86,33 @@ public SkyTab() throws IOException {
   }
 
   @Override public void initialize(URL location, ResourceBundle resources) {
-    simulatedSettings.getChildren().add(horizonOffset);
-    horizonOffset.setName("Horizon offset");
-    horizonOffset.setTooltip("Moves the simulated horizon.");
```

Comment:
"the simulated horizon"… so this didn't affect other skies at all and didn't work for simulated sky?

Reply:
It does do something with the horizon but it's somehow bugged for Preetham and not implemented yet with Nishita. This is more of a temporary fix until both can get fixed/implemented.

Non-zero offset:
![image](https://user-images.githubusercontent.com/3740917/107713039-588ca680-6cc2-11eb-8a49-faf442565e6a.png)

Reply:
So, it the offset did certainly work for Preetham and for whatever reason the copy-paste job of the Preetham code for this PR broke it... Only the situated sky modes use the offset.

0
![image](https://user-images.githubusercontent.com/3740917/107714311-f6817080-6cc4-11eb-9bca-0a0715ebcb92.png)

1
![image](https://user-images.githubusercontent.com/3740917/107714353-07ca7d00-6cc5-11eb-9a1b-085e37fd2342.png)

Reply:
I believe it did work with both `Preetham` and `Nishita`, however it had the weird circle in the middle if I implemented it on the sky cache. I would have to reset the sky cache and re-bake the sky every time you changed horizon offset. I can try implementing that and testing it.

Reply:
I readded it in 727b301. Changing the horizon offset requires resetting the sky cache. It seems to also cause issues in Nishita sky with offset > 0.25 or so.